setup.py

```diff
@@ -26,8 +26,8 @@
 import warnings
 
 
-if sys.version_info[:2] < (3, 6):
-    raise RuntimeError("Python version >= 3.6 required.")
+if sys.version_info[:2] < (3, 7):
+    raise RuntimeError("Python version >= 3.7 required.")
```

Comment:
Is there a reason to actively reject 3.6 or is it enough to declare we no longer support it without forcing an error?

Reply:
Not sure. Last I knew 1.20 still works with 3.6, we just don't want to officially release wheels for it and no longer test it. The minimal change for 1.20 would be to just remove 3.6 from `tox.ini` and leave everything else alone. OTOH, I believe there was a 3.6  incompatible change recently merged into master. So this is probably OK for master.